SampleService.spec

```diff
@@ -273,6 +273,20 @@ module SampleService {
     /* Update a sample's ACLs.  */
      funcdef update_sample_acls(UpdateSampleACLsParams params) returns() authentication required;
 
+    typedef structure {
```

Comment:
Added [documentation](https://github.com/kbase/sample_service/pull/421/commits/224595b5070f5ce442d5c2e3a5f83c92077b1b7d#diff-3d7024e606caf64bc713b81040eb3f8e4cf7ce07a0f872e37cbf7fc864bac62eR32-R34). Please let me know if more should be added:

Reply:
That said, it may be worth just deprecating `update_sample_acls` entirely in favor of this method (see my other comment). In which case my suggested name above would be silly.